src/index.js

```diff
@@ -144,8 +150,14 @@ class Dotenv {
       return obj
     }, {})
 
-    // fix in case of missing
-    formatted['process.env'] = '{}'
+    // We have to stub any remaining `process.env`s due to Webpack 5 not polyfilling it anymore
+    // https://github.com/mrsteele/dotenv-webpack/issues/240#issuecomment-710231534
+    // However, if someone targets Node or Electron `process.env` still exists, and should therefore be kept
+    // https://webpack.js.org/configuration/target
+    if (!target.startsWith('node') && !target.startsWith('electron')) {
+      // Results in `"MISSING_ENV_VAR".NAME` which is valid JS
+      formatted['process.env'] = '"MISSING_ENV_VAR"'
```

Comment:
If you already omitting it for node and electron why is there a need for changing it to `"MISSING_ENV_VAR"`? We could still use `{}` because it was only breaking for setters which are not existing in browser environments.

Reply:
It is erroring even when just accessing `{}.NAME` for us. We had to update our own stub to this, as well as opening this PR or it will give us the famous `Unexpected character '.'`.

I checked and it is indeed just accessing `{}` that is breaking, and specifically only when this plugin does its replacement.

Reply:
Interesting, you're right. Well, then we have to use this workaround I guess.

Another solution would be the following but it will not make any different to yours:

```javascript
formatted['process.env'] = 'new Object({})'
```

Reply:
I like this string because it also does explain why we have this weird code left over. Rather than `{}.NAME` it literally says that an environment variable is missing